### ISSUE TITLE
Simple ViewController UI snapshot testing workflow

### ISSUE THREAD

**Issue by cadizjavier** [pointfreeco/swift-snapshot-testing#47]
I'm trying to do a simple UI snapshot testing using a ViewController with some UILabel text centered.

In the test file, I instantiate the ViewController and do an `assertSnapshot`. First time fails as expected and records the snapshot time in the filesystem as expected.
The second time I run it (it should pass) fails logging a mismatch between the sizes of the images.

Error: `failed - Expected image@(375.0, 812.0) to match image@(250.0, 541.333333333333)`

I have 2 issues/concerns here:
- Images in the report are the same but somehow it compares different sizes so it reports a failing test.
- The error log should report what filename is comparing to check manually in the filesystem. In this test, I only have 1 image but it can be trickier to find the proper image if I have several snapshots recorded.

**Comment by stephencelis** [comment]
Somehow we never responded to this issue, sorry! We've done a pretty major infrastructural overhaul. Can you check to see if this is still an issue?

One other question: what simulator are you running your tests on and can you give us a code snippet of the work you're doing in the test?

**Comment by cadizjavier** [comment]
Hi! I have done another try and it works now flawlessly.
I just added a tiny [PR](https://github.com/pointfreeco/swift-snapshot-testing/pull/81) with a fix to an issue I have detected when installing the library and a minor improvement to the README.
The other issue I have encountered regarding the Docs is related to the need to use a XCTest subclass  but it should be fixed with [this](https://github.com/pointfreeco/swift-snapshot-testing/pull/79) one.

Didn't remember the simulator at that time but now I have tested in several ones and all works as expected.
The overall flow is very nice, intuitive and fast.
Thanks for keeping it up with this useful lib.
